File: Artist/app/src/main/java/com/example/artist/network/TopAlbumsService.java
package com.example.artist.network;


import com.example.artist.models.TopAlbumsResponse;
import com.example.artist.models.TopAlbumsResponseResult;

import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Query;


public interface TopAlbumsService {
    @GET("?method=album.search&format=json")
    Single<TopAlbumsResponseResult> getTopArtists(@Query("album") String user, @Query("api_key") String apiKey);

}

File: Artist/app/src/main/java/com/example/artist/network/TopAlbumsService.kt
package com.example.artist.network

import com.example.artist.models.TopAlbumsResponseResult
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface TopAlbumsService {
    @GET("?method=album.search&format=json")
    fun getTopArtists(@Query("album") user: String, @Query("api_key") apiKey: String): Single<TopAlbumsResponseResult>
}